### PR TITLE
Upload approved but not translateable fso datasets

### PR DESCRIPTION
**Pull request by sabinem**
Load the following datasets to postgres and their metadata to postgres and fix their queries if needed (postgres is stricter on the sql sytax then sql lite) 

- datasets from fso that have been approved and can't be translated into English

| Pipeline    | nr queries| approved | languages available | translateable into en |
| --------- | ----- | ------- | ----------- | -------- | 
| A4:heavy_check_mark:  | 1 | :heavy_check_mark:| de | x | 
| A5:heavy_check_mark: | 1 | :heavy_check_mark:| de,fr | x |  |    |  |
| A8 :heavy_check_mark: | 4 |  :heavy_check_mark: | de,fr | x | 
| S1 :heavy_check_mark: | 10 | :heavy_check_mark:| en, de |  x | 
| A17 :heavy_check_mark: | 1 |:heavy_check_mark: |de.fr   |x |

### CODE REVIEW

File: pipelines/S1/queries.log [hunk 1]
```diff
@@ -44,43 +45,146 @@ SELECT DISTINCT verbrauchskategorie, verbrauchskategorie_beschreibung FROM media
...
- jahr verbrauchskategorie MAX(T.mittlerer_preis_rappen_pro_kwh)
- 2024                  H1                              37.78000
- 2024                  H2                              35.61583
- 2024                  H3                              30.68278
- 2024                  H4                              33.49479
- 2024                  H5                              29.46551
- 2024                  H6                              25.50494
- 2024                  H7                              29.58998
- 2024                  H8                              32.65728
+SELECT T.jahr, T.verbrauchskategorie, MAX(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Waadt%'     AND S.canton=TRUE     AND T.energieprodukt='Standardprodukt'     AND T.verbrauchskategorie LIKE 'H_' GROUP BY T.verbrauchskategorie, T.jahr, T.mittlerer_preis_rappen_pro_kwh;
```
Comment (sabinem) [inline pipelines/S1/queries.log:60]: This query has an unexpected result. Not sure how to fix this.

File: pipelines/S1/queries.log [hunk 2]
```diff
@@ -92,26 +196,106 @@ SELECT T.jahr, T.energieprodukt, T.verbrauchskategorie, T.mittlerer_preis_rappen
...
-SELECT T.jahr, T.verbrauchskategorie, MAX(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Zürich%'     AND S.canton=TRUE     AND T.verbrauchskategorie LIKE 'H_%'     AND T.verbrauchskategorie_grosse_kwh_pro_jahr < 7500 GROUP BY T.verbrauchskategorie ORDER BY T.mittlerer_preis_rappen_pro_kwh DESC;
- jahr verbrauchskategorie MAX(T.mittlerer_preis_rappen_pro_kwh)
- 2024                  H1                              40.04015
- 2024                  H2                              37.25029
- 2024                  H4                              35.15730
- 2024                  H3                              33.91238
+SELECT T.jahr, T.verbrauchskategorie, MAX(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Zürich%'     AND S.canton=TRUE     AND T.verbrauchskategorie LIKE 'H_%'     AND T.verbrauchskategorie_grosse_kwh_pro_jahr < 7500 GROUP BY T.verbrauchskategorie, T.mittlerer_preis_rappen_pro_kwh, T.jahr ORDER BY T.mittlerer_preis_rappen_pro_kwh DESC;
```
Comment (sabinem) [inline pipelines/S1/queries.log:216]: This query seems to have similar problems as the one above.

File: pipelines/S1/queries.log [hunk 3]
```diff
@@ -44,43 +45,146 @@ SELECT DISTINCT verbrauchskategorie, verbrauchskategorie_beschreibung FROM media
 Query Nr. 3
 -- Wie viel zahle ich in CHF für 1000 kWh bei einer 2-Zimmer-Wohnung im Kanton Bern im Jahr 2017?
 SELECT T.verbrauchskategorie, T.verbrauchskategorie_beschreibung, T.energieprodukt, T.mittlerer_preis_rappen_pro_kwh * 1000 / 100 as preis_franken_fur_1000_kwh_bern_2017_2_zimmerwohnung FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Bern%'     AND S.canton=TRUE     AND T.jahr=2017     AND T.verbrauchskategorie_beschreibung LIKE '%2-zimmerwohnung%'     AND T.verbrauchskategorie_grosse_kwh_pro_jahr >= 1000;
```
Comment (sabinem) [inline pipelines/S1/queries.log:47]: Here the query was not changed, but result looks unexpected.

File: pipelines/S1/queries.log (hunk 1 above)
Comment (cmdoret) [reply pipelines/S1/queries.log:60]: I had to make the query more complex to accomodate postgres' `group by` logic

File: pipelines/S1/queries.log (hunk 2 above)
Comment (cmdoret) [reply pipelines/S1/queries.log:216]: I also had to make the query more complex for the `group by` to work

File: pipelines/S1/queries.log (hunk 3 above)
Comment (cmdoret) [reply pipelines/S1/queries.log:47]: This was because the `LIKE` function is case-insensitive in SQLite, but case sensitive in postgreSQL :(

**Review by cmdoret — APPROVED** [review]

**Pull request merged**